lib/rules/applet-alt.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "applet-alt",
+  "selector": "applet",
+  "matches": "",
+  "tags": [
+    "cat.text-alternatives",
+    "wcag2a",
+    "wcag111"
+  ],
+  "metadata": {
+    "description": "Checks if a text alternative is provided for an applet",
+    "help": "Label an applet by using the alt/labelledby/label text alternative and provide a text alternative in the body of the applet element "
```

Comment:
The help text goes in the sidebar of the aXe extensions, so this needs to be shorter. You could swap description and help text to accomplish this. See my note about the active tense.